proposed/cache-meta.md

```diff
@@ -279,16 +279,19 @@ do so.
 5. People
 ---------
 
-### 5.1 Editor
+### 5.1 Coordinator
+* Paul Dragoonis
+
+### 5.2 Editor
 
 * Larry Garfield
 
-### 5.2 Sponsors
+### 5.3 Sponsors
 
-* Beau Simensen, Sculpin (Coordinator)
+* Beau Simensen, Sculpin
 * Pádraic Brady, Zend Framework
 
-### 5.3 Contributors
+### 5.4 Contributors
 
 * Paul Dragoonis
```

Comment:
Per the blyaw, Editor, Coordinator, and Sponsor should not be listed in the Contributors list.